text/0097-log-data-model.md

```diff
@@ -0,0 +1,1005 @@
+defined but the `SeverityText` is missing it is recommended to only show the
+short name, e.g. **INFO**.
+
+When drop down lists (or other UI elements that are intended to represent the
+possible set of values) are used for representing the severity it is preferable
+to display the short name in such UI elements.
+
+For example a dropdown list of severities that allows filtering log records by
+severities is likely to be more usable if it contains the short names of
+`SeverityNumber` (and thus has a limited upper bound of elements) compared to a
+dropdown list, which lists all distinct `SeverityText` values that are known to
+the system (which can be a large number of elements, often differing only in
+capitalization or abbreviated, e.g. "Info" vs "Information").
+
+#### Comparing Severity
+
+In the contexts where severity participates less-than / greater-than comparisons
+`SeverityNumber` field should be used. `SeverityNumber` can be compared to
+another `SeverityNumber` or to numbers in the 1..24 range (or to the
+corresponding short names).
+
+When severity is used in equality or inequality comparisons (for example in
+filters in the UIs) the recommendation is to attempt to use both `SeverityText`
+and short name of `SeverityNumber` to perform matches. For example if we have a
+record with `SeverityText` field equal to "Informational" and `SeverityNumber`
+field equal to INFO then it may be preferable from the user experience
+perspective to ensure that **severity="Informational"** and **severity="INFO"**
+conditions both to are TRUE for that record.
+
+### Field: `ShortName`
+
+Type: string.
+
+Description: Short event identifier that does not contain varying parts.
+`ShortName` describes what happened (e.g. "ProcessStarted"). Recommended to be
+no longer than 50 characters. Optional. Not guaranteed to be unique in any way.
+Typically used for filtering and grouping purposes in backends.
+
+### Field: `Body`
+
+Type: any.
+
+Description: A value containing the body of the log record (see the description
+of `any` type above). Can be for example a human-readable string message
+(including multi-line) describing the event in a free form or it can be a
+structured data composed of arrays and maps of other values. Can vary for each
+occurrence of the event coming from the same source.
+
+### Field: `Resource`
+
+Type: key/value pair list.
+
+Description: Describes the source of the log, aka
+[resource](https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/overview.md#resources).
+"value" of each pair is of `any` type. Multiple occurrences of events coming
+from the same event source can happen across time and they all have the same
+value of `Resource`. Can contain for example information about the application
+that emits the record or about the infrastructure where the application runs.
+Data formats that represent this data model may be designed in a manner that
```

Comment:
Confused by this - isn't this PR a "data format"? If the spec says that resource is a part of log record, that invites repetition.

Reply:
This PR is a "data model". It defines semantics of what data constitutes a log record/ The data model specification is human readable, it does not define a machine readable data format.

The data model may be represented in many different ways. For example we can represent it as a JSON, as YAML or as a ProtoBuf message. Such representation will define a machine readable "data format".